naccs_bot.py

```diff
@@ -159,69 +160,73 @@ def get_ongoing_matches(channel_id):
-    image = (jsonstreams["payload"][x]["stream"]["channelLogo"])
-    competitionName = (jsonstreams["payload"][x]["competitionName"])
-    channelUrl = (jsonstreams["payload"][x]["stream"]["channelUrl"])
+        nick = (jsonstreams["payload"][x]["userNickname"])
+        image = (jsonstreams["payload"][x]["stream"]["channelLogo"])
+        competitionName = (jsonstreams["payload"][x]["competitionName"])
+        channelUrl = (jsonstreams["payload"][x]["stream"]["channelUrl"])
+        if (toggle == 1):
+            activestreams[nick] = nick
+        else:
+            activestreams2[nick] = nick
+        if nick in dispstreams:
+            continue
+        else:
+            channel = client.get_channel(LEAGUE_STREAMS)
+            embed = Embed(title=nick, url=channelUrl, description=competitionName, color=0x5e7aac)
+            embed.set_author(name="A League Stream is Live!", icon_url="https://naccs-s3.s3.us-east-2.amazonaws.com/static/assets/headerlogo_small.png")
+            embed.set_thumbnail(url=image)
+            activeembed = await channel.send(embed=embed)
+            dispstreams[nick] = activeembed.id
+    # From here down is checking to see if our active displayed streamers are still in the api, if not, we will delete their messages.
     if (toggle == 1):
-        activestreams[nick] = nick
-    else:
-        activestreams2[nick] = nick
-    if nick in dispstreams:
-        # No need for another embed if they already have one.
-        continue
+        overactive = activestreams
     else:
-        # Creating an embed for the streamer
-        channel = client.get_channel(LEAGUE_STREAMS)
-        embed = Embed(title=nick, url=channelUrl, description=competitionName, color=0x5e7aac)
-        embed.set_author(name="A League Stream is Live!", icon_url="https://naccs-s3.s3.us-east-2.amazonaws.com/static/assets/headerlogo_small.png")
-        embed.set_thumbnail(url=image)
-        activeembed = await channel.send(embed=embed)
-        dispstreams[nick] = activeembed.id
-    # From here down is checking to see if our active displayed streamers are still in the api, if not, we will delete their messages.
-  if (toggle == 1):
-      overactive = activestreams
-  else:
-      overactive = activestreams2
-  over = { k : dispstreams[k] for k in set(dispstreams) - set(overactive) }
-
-  if over:
-      for key in over:
-        channel = client.get_channel(LEAGUE_STREAMS)
-        msg = await channel.fetch_message(dispstreams[key])
-        await msg.delete()
-        del dispstreams[key]
-        over = {}
+        overactive = activestreams2
+    over = { k : dispstreams[k] for k in set(dispstreams) - set(overactive) }
+    if over:
+        for key in over:
+            channel = client.get_channel(LEAGUE_STREAMS)
+            msg = await channel.fetch_message(dispstreams[key])
+            await msg.delete()
+            del dispstreams[key]
+            over = {}
+
+    if (toggle == 1):
```

Comment:
This isn't supposed to be here right? Since run_get_streams will be looped by discord.ext.tasks?

Reply:
Will be removed with new api response iteration.